AndroidAnnotations/androidannotations-rest-spring/rest-spring-api/src/main/java/org/androidannotations/rest/spring/annotations/Rest.java

```diff
@@ -170,6 +170,47 @@
  * </pre>
  * 
  * </blockquote>
+ *
+ * <h1>ResponseErrorHandler</h1>
+ * <p>
+ * You can use your own error handler if you want to customize how errors
+ * are handled. The {@link #responseErrorHandler()} parameter lets you define the
+ * {@link org.springframework.web.client.ResponseErrorHandler
+ * ResponseErrorHandler}.
```

Comment:
You can use your own error handler to customize how errors
+ * are handled. The {@link #responseErrorHandler()} parameter lets you define the
+ * {@link org.springframework.web.client.ResponseErrorHandler
+ * ResponseErrorHandler}.